src/stories/5-custom/filter/filter-checkbox-item.stories.tsx

```diff
@@ -40,20 +40,65 @@ export default {
 			},
 			type: { name: "object", value: {} },
 		},
+		collapsible: {
+			description: "Specifies if the contents can be collapsed or expanded",
+			control: {
+				type: "boolean",
+			},
+			defaultValue: true,
+		},
+		showDivider: {
+			description: "Specifies if header divider is visible in default mode",
+			control: {
+				type: "boolean",
+			},
+			defaultValue: true,
+		},
+		showMobileDivider: {
+			description: "Specifies if divider is visible in mobile mode",
+			control: {
+				type: "boolean",
+			},
+			defaultValue: true,
+		},
+		defaultValues: {
+			description: "Default value for the field, this is declared outside `sections`",
+			table: {
+				type: {
+					summary: "string[]",
+				},
+			},
+			type: { name: "object", value: {} },
+		},
```

Comment:
move `defaultValues` argType to `FilterCheckBoxItemWithDefaultValues` story because it is not part of the schema and only used in that story.

```
FilterCheckBoxItemWithDefaultValues.argTypes = {
	defaultValues: {
		description: "Default value for the field, this is declared outside `sections`",
		table: {
			type: {
				summary: "string[]",
			},
		},
		type: { name: "object", value: {} },
	},
};
```